src/library/scala/collection/TraversableOnce.scala

```diff
@@ -402,8 +402,8 @@ trait TraversableOnce[+A] extends Any with GenTraversableOnce[A] {
 
 
 object TraversableOnce {
-  implicit def alternateImplicit[A](trav: TraversableOnce[A]) = new ForceImplicitAmbiguity
-  implicit def flattenTraversableOnce[A, CC[_]](travs: TraversableOnce[CC[A]])(implicit ev: CC[A] => TraversableOnce[A]) =
+  implicit def alternateImplicit[A](trav: TraversableOnce[A]): ForceImplicitAmbiguity = new ForceImplicitAmbiguity
```

Comment:
For implicits, yes, for some TBD value of "yes". Currently, type inference and implicit search cycles can cause very hard to understand implicit search ambiguities. If we can come up with some easy to understand scheme that softens this requirement, great! Otherwise, I'm on board with the change in principle.